tests/trx_finality_status_forked_test.py

```diff
@@ -183,6 +183,7 @@ def getBlockID(status):
         blockNum = getBlockNum(retStatus)
         if state == forkedOutState or ( info['head_block_producer'] == 'defproducerd' and info['last_irreversible_block_num'] > blockNum ):
             break
+        prodD.waitForNextBlock()
```

Comment:
This was not needed, but just noticed the test was spamming the node with requests. A sleep would have worked here as well.